tests/js/spec/components/modals/addDashboardWidgetModal.spec.jsx

```diff
@@ -1411,9 +1409,7 @@ describe('Modals -> AddDashboardWidgetModal', function () {
       });
 
       await tick();
-      await act(async () =>
-        userEvent.click(screen.getByLabelText('Metrics (Release Health)'))
-      );
+      await act(async () => userEvent.click(screen.getByLabelText(/metrics/i)));
```

Comment:
so we need act here?

Reply:
Looks like we do, because of changes to MetricsSearchBar.